src/sentry/static/sentry/app/views/organizationIntegrations/installedPlugin.tsx

```diff
@@ -82,19 +98,25 @@ export class InstalledPlugin extends React.Component<Props> {
     });
   };
 
-  enablePlugin = async () => {
+  toggleEnablePlugin = async (projectId: string, status: boolean = true) => {
     try {
       addLoadingMessage(t('Enabling...'));
-      await this.pluginUpdate({enabled: true});
-      addSuccessMessage(t('Configuration was enabled'));
-      this.props.onEnablePlugin(this.projectId);
+      await this.updatePluginEnableStatus(status);
+      addSuccessMessage(
+        tct('Configuration was [action].', {action: status ? 'enabled' : 'disabled'})
```

Comment:
@NisanthanNanthakumar The words `enabled` and `disabled` should be translated here so we shouldn't use variable substitution like this. Just have the redundancy of calling `t` two different ways.